cli/integration_tests/turbo_help.t

```diff
@@ -5,203 +5,204 @@ Test help flag
+  
+  Run Arguments:
+        --cache-dir <CACHE_DIR>          Override the filesystem cache directory
+        --cache-workers <CACHE_WORKERS>  Set the number of concurrent cache operations (default 10) [default: 10]
+        --concurrency <CONCURRENCY>      Limit the concurrency of task execution. Use 1 for serial (i.e. one-at-a-time) execution
+        --continue                       Continue execution even if a task exits with an error or non-zero exit code. The default behavior is to bail
+        --dry-run [<DRY_RUN>]            [possible values: stdout, text, json]
+        --filter <FILTER>                Use the given selector to specify package(s) to act as entry points. The syntax mirrors pnpm's syntax, and additional documentation and examples can be found in turbo's documentation https://turbo.build/repo/docs/reference/command-line-reference#--filter
+        --force                          Ignore the existing cache (to force execution)
+        --global-deps <GLOBAL_DEPS>      Specify glob of global filesystem dependencies to be hashed. Useful for .env and files
+        --graph [<GRAPH>]                Generate a graph of the task execution and output to a file when a filename is specified (.svg, .png, .jpg, .pdf, .json, .html). Outputs dot graph to stdout when if no filename is provided
+        --ignore <IGNORE>                Files to ignore when calculating changed files (i.e. --since). Supports globs
+        --include-dependencies           Include the dependencies of tasks in execution
+        --no-cache                       Avoid saving task results to the cache. Useful for development/watch tasks
+        --no-daemon                      Run without using turbo's daemon process
+        --no-deps                        Exclude dependent task consumers from execution
+        --output-logs <OUTPUT_LOGS>      Set type of process output logging. Use "full" to show all output. Use "hash-only" to show only turbo-computed task hashes. Use "new-only" to show only new output with only hashes for cached tasks. Use "none" to hide process output. (default full) [default: full] [possible values: full, none, hash-only, new-only, errors-only]
+        --parallel                       Execute all tasks in parallel
+        --profile <PROFILE>              File to write turbo's performance profile output into. You can load the file up in chrome://tracing to see which parts of your build were slow
+        --remote-only                    Ignore the local filesystem cache for all tasks. Only allow reading and caching artifacts using the remote cache
+        --scope <SCOPE>                  Specify package(s) to act as entry points for task execution. Supports globs
+        --since <SINCE>                  Limit/Set scope to changed packages since a mergebase. This uses the git diff ${target_branch}... mechanism to identify which packages have changed
+        --single-package                 Run turbo in single-package mode
```

Comment:
I don't love having all of the run flags present in the root help text. I know they can be passed at root since we allow people to drop `run`, but it really clutters the output.

Reply:
Yeah...I don't know if we can remove them for the root help screen but make them visible for the command help screen. I'll investigate that.